Eatery/Analytics.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Analytics.swift
+//  AppDevAnalytics
+//
+//  Created by Kevin Chan on 9/4/19.
+//  Copyright © 2019 Kevin Chan. All rights reserved.
+//
+
+import Crashlytics
+
+class AppDevAnalytics {
+
+    static let shared = AppDevAnalytics()
+
+    private init() {}
+
+    func log(_ payload: Payload) {
+        #if !DEBUG
+        let fabricEvent = payload.convertToFabric()
+        Answers.logCustomEvent(withName: fabricEvent.name, customAttributes: fabricEvent.attributes)
+        #endif
+    }
+}
+
+    // MARK: Event Payloads
+
+    /// Log whenever Collegetown pill button is pressed
```

Comment:
why are these indented?

Reply:
From transit:

> // MARK: Event Payloads
>
>// MARK: Important
>/// Log app launch with device info
>struct AppLaunchedPayload: Payload {
>    static let eventName: String = "App Launched"
>    let deviceInfo = DeviceInfo()
>}

I thought a MARK: Important was unnecessary since MARK: Event Payloads is more descriptive. Also, I feel like MARKs are “titles” and so I like to give them some indentation instead of tacking on the events right after because it is significant and should be visually separated (so it doesn’t blend in if I’m scrolling through a file looking for something). The same reason applies for why each event has an indentation between them (they’re each significant). I’m assuming you’re asking because you feel the opposite way; how come?

Reply:
This reasoning is similar to if you have two classes in the same file. It would look like:
```
class {

}

class B {

}
```

Reply:
Right; so what point are you trying to make?

Reply:
Wait I might be tripping but right now I see your changes looks like:
```
    struct A {
    }
    
    struct B {
    }
```
when it should look like
```
struct A {
}
    
struct B {
}
```

Reply:
Nope I’m tripping. That probably happened bc I copy pasted it in from a previous branch. Don’t understand why it always tends to indent every time you paste. I’ll fix it ASAP